index.html

```diff
@@ -1682,23 +1682,24 @@ <h3>Validity Period</h3>
         <dl>
           <dt><var>validFrom</var></dt>
           <dd>
-A <a>credential</a> MUST have an <code>validFrom</code> <a>property</a>. The
-value of the <code>validFrom</code> <a>property</a> MUST be a string value of
-an [<a data-cite="XMLSCHEMA11-2#dateTime">XMLSCHEMA11-2</a>] combined
-<code>date-time</code> string representing the date and time the
+A <a>credential</a> MUST have a <code>validFrom</code> <a>property</a>. The
```

Comment:
What happens when this MUST is not enforced by an issuer?

What error does the verify raise?

Reply:
> What happens when this MUST is not enforced by an issuer?

We answer that question here:

https://w3c.github.io/vc-data-model/#conformance

with this text:

> A conforming processor is any algorithm realized as software and/or hardware that generates or consumes a [conforming document](https://w3c.github.io/vc-data-model/#dfn-conforming-document). Conforming processors MUST produce errors when non-conforming documents are consumed.

> What error does the verify raise?

I'm saying this w/o a value judgement... at present, we don't say, on purpose:

> This specification makes no normative statements with regard to the conformance of roles in the ecosystem, such as [issuers](https://w3c.github.io/vc-data-model/#dfn-issuers), [holders](https://w3c.github.io/vc-data-model/#dfn-holders), or [verifiers](https://w3c.github.io/vc-data-model/#dfn-verifier), because the conformance of ecosystem roles are highly application, use case, and market vertical specific.

The group tried to define the algorithm for VC verification and WG members disagreed on when what types of errors should be raised. A lot of time has passed since we had that previous discussion, and I would expect that (if we had the time), we could get to some sort of algorithm with some sort of set of errors defined. That said, given the short time we have until the CR phase, having that discussion now could eat up a ton of time.

So, for right now, we just say that a processor has to raise an error and we're silent on what that error should be.